src/org/thoughtcrime/securesms/ConversationItem.java

```diff
@@ -643,47 +640,45 @@ private void setReactions(final MessageRecord messageRecord) {
   private String getMessageSenderName(Address senderAddress) {
     String messageSenderName = "";
 
-    IdentityDatabase identityDatabase  = DatabaseFactory.getIdentityDatabase(context);
-    Address myAddress = null;
+    IdentityDatabase identityDatabase = DatabaseFactory.getIdentityDatabase(context);
+    Address          myAddress        = null;
 
     try {
-        myAddress = identityDatabase.getMyIdentity().getAddress();
+      myAddress = identityDatabase.getMyIdentity().getAddress();
     } catch (Exception e) {
-        e.printStackTrace();
+      e.printStackTrace();
     }
 
     if (myAddress.serialize().equals(senderAddress.serialize())) {
```

Comment:
@veken1199 This part is a bit messy with all of the if-else's. It can be cleaned up like this:
```
 if (myAddress.serialize().equals(senderAddress.serialize())) return "Me";

 messageSenderName = messageRecord.getRecipient().getName();

 if (messageRecord.getRecipient().getAddress().isGroup()) {
   for (Recipient participant : messageRecord.getIndividualRecipient().getParticipants()) {
     if (participant.getAddress().serialize().equals(senderAddress.serialize())) {
       return participant.getName() == null ? participant.getAddress().serialize() : participant.getName();
     }
   }
 }

 if (messageSenderName == null) {
   messageSenderName = messageRecord.getRecipient().getAddress().toString();
 }

return messageSenderName;
```